cmd/monaco/runner/runner.go

```diff
@@ -61,8 +51,20 @@ func BuildCmd(fs afero.Fs) *cobra.Command {
 	return BuildCmdWithLogSpy(fs, nil)
 }
 
+func writeSupportArchive(fs afero.Fs) func() {
+	return func() { // writing the support archive is a deferred function call in order to guarantee that a support archive is also written in case of a panic
```

Comment:
Old comment from copy/paste still here. Please remove it.

Reply:
removed